.github/workflows/export-tmt-tests-to-polarion.yml

```diff
@@ -0,0 +1,93 @@
+name: Export tmt tests to Polarion work items
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        type: boolean
+        default: false
+        description: Export preview only
+
+jobs:
+  pr-info:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Query author repository permissions
+        uses: octokit/request-action@v2.x
+        id: user_permission
+        with:
+          route: GET /repos/${{ github.repository }}/collaborators/${{ github.event.sender.login }}/permission
```

Comment:
Should this be `github.actor` instead of `github.event.sender.login`? From what I'm seeing in the docs the `sender.login` isn't for workflow_dispatch and `github.actor` will get the user that triggered the workflow